docker/bedrook/start-bedrock

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+
+export NODE=ohio
+export DB=/db/bedrock.db
+export CFG=/db/bedrock.cfg
+export PEERS=/db/bedrock.peers
+export PRIO=30
+export RUN=/var/run/bedrock.pid
+
```

Comment:
Perhaps the above configs should be exposed as ENV vars in the dockerfile? At least the `NODE` name?